pkgs/tools/networking/simple-dlna-browser/default.nix

```
@@ -0,0 +1,54 @@
+{ pkgs, lib, makeWrapper, ...}:
+let
+  buildInputs = with pkgs; [
+    gnused
+    socat
+  ];
+in
+pkgs.stdenv.mkDerivation {
+  pname = "simple-dlna-browser";
+  version = "unstable-2023-01-09";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "javier-lopez";
+    repo = "learn";
+    sparseCheckout = [
+      "sh/tools/simple-dlna-browser"
+    ];
+    rev = "666e2c402723251ba19ce9d7c11b83b06c15a2c5";
+    sha256 = "sha256-jwPAEgti5DrjNnNVW/61GGTP+QedGAjz9kPwYhMAdTk=";
+  };
+
+  inherit (buildInputs);
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    outputWrapper="$out/bin/simple-dlna-browser"
+    outputWrapped="$out/bin/wrapped-simple-dlna-browser"
+
+    mkdir -p $out/bin
+    cp sh/tools/simple-dlna-browser $outputWrapped
+    makeWrapper $outputWrapped $outputWrapper \
+      --prefix PATH : ${lib.makeBinPath buildInputs}
```

If they are only used for wrapping here then buildInputs above is doing nothing and we can simplify this to 

```suggestion
      --prefix PATH : ${lib.makeBinPath [ gnused socat ]}
```

---

I just tried this, but got `error: undefined variable 'gnused'`. How does one access these packages without using `pkgs` ?

---

Ah... I get it now. `callPackage` uses dependency injection. We desperately need documentation on the pattern as well as to generate documentation from nix code like `javadoc` and `cargo doc`...

Anyway, this should work now in 268feff